tests/test_create_analysis_json.py

```diff
@@ -70,8 +72,37 @@ def test_get_format(self):
         self.assertEqual(analysis_json.get_format('asdf.bam', {'.bam': 'bam', '_metrics': 'metrics'}), 'bam')
         self.assertEqual(analysis_json.get_format('asdf.foo_metrics', {'.bam': 'bam', '_metrics': 'metrics'}), 'metrics')
 
+    @requests_mock.mock()
+    def test_create_core_success(self, mock):
+        type = 'analysis'
+        schema_version = 'good_version'
+        schema_url = 'https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/{0}/json_schema/analysis.json'.format(schema_version)
+
+        mock.head(schema_url, status_code=200)
+        core = analysis_json.create_core(type=type, schema_version=schema_version)
+        expected_core = {
+            'type': type,
+            'schema_url': schema_url,
+            'schema_version': schema_version
+        }
+        self.assertEquals(core, expected_core)
+
+    @requests_mock.mock()
+    def test_create_core_failure(self, mock):
+        type = 'analysis'
+        schema_version = 'bad_version'
+        schema_url = 'https://raw.githubusercontent.com/HumanCellAtlas/metadata-schema/{0}/json_schema/analysis.json'.format(schema_version)
+
+        mock.head(schema_url, status_code=404)
+
+        try:
+            core = analysis_json.create_core(type=type, schema_version=schema_version)
+        except HTTPError:
```

Comment:
Could we update the test to fail if an error is not raised? Right now it looks like it would pass either way